client/src/cmdlfnexwatch.c

```diff
@@ -108,6 +109,21 @@ static int nexwatch_scamble(NexWatchScramble_t action, uint32_t *id, uint32_t *s
     return PM3_SUCCESS;
 }
 
+static int nexwatch_magic_bruteforce(uint32_t cn, uint8_t calc_parity, uint8_t chk) {
+    uint8_t magic = 0;
+    uint8_t temp_checksum;
+    for (; magic < 255; magic++) {
```

Comment:
cppchecker will say that you can lessen the scope of these two vars.   ie,  move them inside.

Reply:
do you mean we can remove the "static" ?